test/e2e/lib/types.ts

```diff
@@ -1,3 +1,6 @@
 // Browser Manager
 export type screenSize = 'desktop' | 'mobile' | 'laptop' | 'tablet';
 export type localeCode = string;
+
+// General
+export type timeoutMS = number;
```

Comment:
nit: this is simple enough that you really don't have to export it as a custom type -- you could just do `config.get() as number`

Reply:
Sure, will remove in next commit.

Reply:
To be clear, the reason this doesn't help in TypeScript is because it is just an alias. In any place where you expect a `timeoutMS` you can pass a `number` and TS won't complain.